src/config/dynamic-theme-fixes.config

```diff
@@ -1190,30 +1190,46 @@ img[src*="https://github.githubassets.com/favicon.ico"].smart-link-icon
 img[src^="https://latexmath.bolo-app.com/render/"]
 
 CSS
-#jira-issue-header, #jira-issue-header-actions {
-    background-color: transparent !important;
-}
 span.code {
     background-color: rgba(240,246,252,0.15) !important; 
+    padding: 2px 3px !important;                                                                
```

Comment:
Uhm what? We don't do cosmetic CSS like padding.

Reply:
Removed it in commit 73eef25